com/win32comext/taskscheduler/test/test_addtask_1.py

```diff
@@ -33,12 +33,12 @@
 )
 new_task.SetIdleWait(1, 10000)
 new_task.SetComment("test task with idle trigger")
-new_task.SetApplicationName("c:\\python23\\python.exe")
+new_task.SetApplicationName("c:\\python37\\python.exe")
```

Comment:
maybe "py.exe" with no path makes sense here and below, and omit the `SetWorkingDirectory` call?

Reply:
If the tests are still valid, sounds good to me.

Reply:
I'd not be at all surprised if these went stale somewhere since python 2.3, but the rest of the test infrastructure tends to use `py.exe` so yeah, let's do that here and we can worry about whether it actually still works later.